src/Libraries/CoreNodeModels/Input/BasicInteractive.cs

```diff
@@ -50,6 +51,18 @@ public override string PrintExpression()
             return Value.ToString();
         }
 
+        public override bool IsConvertible
```

Comment:
Can NodeModel class have a constructor to set convertible and can't the base class implement GetAstIdentifierForOutputIndex()?